File: 04-19 - Grafos/tads/graph/edge.cpp
#pragma once

class Edge
{
public:
  int from;
  int to;
  int cost;

  Edge(int from, int to, int cost)
  {
    this->from = from;
    this->to = to;
    this->cost = cost;
  }

  Edge(int from, int to)
  {
    this->from = from;
    this->to = to;
    this->cost = 0;
  }
};

File: 04-19 - Grafos/tads/graph/Edge.java
package tads.graph;

public class Edge {
  public int from;
  public int to;
  public int cost;

  public Edge(int from, int to) {
    this.from = from;
    this.to = to;
    this.cost = 1;
  }

  public Edge(int from, int to, int cost) {
    this.from = from;
    this.to = to;
    this.cost = cost;
  }
}
